src/devices/bus/isa/svga_paradise.cpp

```diff
@@ -329,6 +358,14 @@ ROM_START( wd90c30_lr )
 	ROM_REGION(0x8000,"vga_rom", ROMREGION_ERASE00)
 	ROM_SYSTEM_BIOS(0, "wd90c30_lr", "Western Digital WD90C30-LR")
 	ROMX_LOAD(  "90c30-lr.vbi", 0x000000, 0x008000, CRC(3356ad43) SHA1(6cd56cf274b3c9262b7ca12d49bae63afc331c58), ROM_BIOS(0) )
+	// WDC 90C30 reference design - Chips: WDC WD90C30-LR, MUSIC TR9C1710-80DCA - ROMS: 62-003265-008 - RAM: 512KB, up to 1MB
+	// Connectors: DB15, 26pin EDGE - DIP: 5 way (SW1)
+	// Identical ROMs are used on a VG-8000 WD90C30 VGA
+	ROM_SYSTEM_BIOS(1, "wdc30lr_hl", "WD90C30-LR Hi/Lo")
+	ROMX_LOAD("u24.bin", 0x000000, 0x008000, CRC(797ec96d) SHA1(902418504a446f0a484fd3da6f65d3cff3987280), ROM_SKIP(1) | ROM_BIOS(1) )
+	ROM_IGNORE(					   0x008000 )
+	ROMX_LOAD("u24.bin", 0x000000, 0x008000, CRC(797ec96d) SHA1(902418504a446f0a484fd3da6f65d3cff3987280), ROM_SKIP(1) | ROM_BIOS(1) )
+	ROM_IGNORE(					   0x008000 )
 ROM_END
```

Comment:
I'm not sure what's the goal here, is `u24.bin` really supposed to ROM_SKIP in the first place?